opensafely/pull.py

```diff
@@ -0,0 +1,73 @@
+import subprocess
+import sys
+
+
+DESCRIPTION = (
+    "Command for updating the docker images used to run OpenSAFELY studies locally"
+)
+REGISTRY = "ghcr.io/opensafely-core"
+IMAGES = ["r", "python", "jupyter", "stata-mp"]
```

Comment:
Should we be getting these out of jobrunner instead? It would be very easy to add a new image there and forget to support it here.

Reply:
Oh, good idea.

Reply:
Hoo boo - I forgot about cohortextractor!

Very good call :D